### DISCUSSION TITLE
导出导入时不传递评论时间

### DISCUSSION THREAD

**Discussion by Nemo1166** [walinejs/waline#1442] [Q&A]
如题，导入 json 时全部导入的评论时间都变成了导入时的时间。

![image](https://user-images.githubusercontent.com/38162382/192942761-ddf96f9b-45f0-4712-9493-f18deb59fb68.png)

**Comment by lizheming** [comment]
使用的是什么存储服务？

**Reply by Nemo1166** [reply]
vercel + leancloud 国际版

Edit: 补充一下，导入端是独立部署 + MySQL

**Reply by rockbenben** [reply]
> 使用的是什么存储服务？

配置环境和问题都与楼主相同。将数据导入到 Vercel 和 LeanCloud 国际版的时间是正常的。是独立部署或Mysql需要设置其他环节吗？

**Comment by rockbenben** [comment]
原因找到了。

`2022-12-11T23:10:28.121Z` 这个时间戳中的格式 "T" 和 "Z" 不被 MySQL 支持，要将 "T" 替换为空格并将 "Z" 删除。

但还原 json 时间依然会重置，我是通过在 mysql 用 INSERT INTO 来手动导入解决的。
